javascript/javascript1/week3/arrayRemoval.js

```diff
@@ -0,0 +1,17 @@
+//Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+
+  const nameToRemove = names.splice(1,1);
```

Comment:
Your code will only remove "Ahmad" if it is the second item in the array. If I run this code with this array will the name "Ahmad" be removed?
```
const names = [
    "Peter",
    "Yana",
    "kristina",
    "Rasmus",
    "Ahmad",
    "Samuel",
    "katrine",
    "Tala",
  ];
```
You need to add code to find where "Ahmad" is in the array and then remove it. Try `const ahmadIndex = names.indexOf("Ahmad");`

Reply:
thanks i will rearrange it